### DISCUSSION TITLE
Full Text Search - Multi Language Field

### DISCUSSION THREAD

**Discussion by younes-alouani** [prisma/prisma#14720] [Q&A]
Hello dear community, 

I would like to use Prisma's "Full Text Search" feature for the following model: 

```
model Blog {
  id      Int    @unique
  content String
  title   String
  @@fulltext([content])
}
```
Knowing that users can create blogs in Spanish or English. 

I want to index the blogs by title and content, then show search results based on the user cookies (spanish or english accepted language). 

Prisma's documentation explains how to create an index and search but it doesn't explain the case where I need to create two indexes (one index per language). 

I want to do something like this with Prisma : 
```
CREATE INDEX blog_idx ON blog USING GIN (to_tsvector('english', content));
CREATE INDEX blog_idx ON blog USING GIN (to_tsvector('spanish', content));
```
```
SELECT * FROM blog
WHERE to_tsvector('english', content)
      @@ to_tsquery('english', 'cars in brazil');
```

**Comment by austincrim** [comment]
Hey @younes-alouani 👋,

This kind of configuration is not possible with Prisma. Would you mind opening a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=)?

In the meantime, you could try [customizing the generated migrations](https://www.prisma.io/docs/guides/database/developing-with-prisma-migrate/include-unsupported-database-features#customize-a-migration-to-include-an-unsupported-feature).

**Comment by younes-alouani** [comment]
Hello, 
Thank you for the proposal. 
However, it is not feasible, as my company wants to implement this feature as soon as possible. 
So I will do the configuration manually using raw queries. It's slow but I have no choice. 
Kind regards,

**Reply by younes-alouani** [reply]
It Looks like this feature is already in the prisma roadmap : https://prismaio.notion.site/Prisma-Roadmap-50766227b779464ab98899accb98295f?p=6e77c1dd72dc46d092289aa054569895&pm=s

**Reply by Jolg42** [reply]
Hi @younes-alouani, we are actually designing FTS to include the `language` like you mentioned.
If you could take a couple of minutes to fill this survey, it would be helpful!
-> https://prisma103696.typeform.com/fts-survey

**Reply by younes-alouani** [reply]
Hi @Jolg42 , 
It's done :) !

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
